script/gen_requirements_all.py

```diff
@@ -61,6 +61,8 @@
     os.path.dirname(__file__), "../homeassistant/package_constraints.txt"
 )
 CONSTRAINT_BASE = """
+# Constrain pillow to 8.2.0 because later versions are causing issues in nightly builds.
+
```

Comment:
We can keep the constraint with the warning if we adjust line 188 with `reqs = set(x for x in integration.requirements if x not in CONSTRAINT_BASE)`:

```suggestion
# Constrain pillow to 8.2.0 because later versions are causing issues in nightly builds.
pyllow==8.2.0
```

Reply:
I don't think we should do that in this PR. This is just a revert PR.

Reply:
Ok sorry. I can add in a followPR.
Feel free to revert my comments request.